build_your_own_images.md

```diff
@@ -0,0 +1,185 @@
+# Building Docker images that contain the Kong Gateway
+
+Kong Software uses the [docker-kong github
+repository](https://github.com/Kong/docker-kong/) to build Docker images that
+contain the Kong Gateway. We will use  
+[docker-kong](https://github.com/Kong/docker-kong/) as a reference
+implementation for describing how to build your own Docker images that contain the Kong
+Gateway.
+
+To build your Docker image, you will need to provide 
+
+1. A base image of your choice
+1. An entrypoint script that runs the Kong Gateway
+1. A Dockerfile that installs the Kong Gateway from a location you specify
+
+
+## Base image
+Use whatever your org makes you use. Kong Software pushes `.apk`, `.deb`, and
+`.rpm` packages to our [public package repository](https://download.konghq.com/).
+
+## Entrypoint script
+
+Get the [entrypoint
+script](https://raw.githubusercontent.com/Kong/docker-kong/master/docker-entrypoint.sh)
+from the [docker-kong github repository](https://github.com/Kong/docker-kong/) and put it in
+directory where you are planning to run the command to build your Docker image.
+
+## Create a Dockerfile to install Kong Gateway
+
+### Decide how to get the Kong Gateway package
+Kong Software provides `.apk`, `.deb`, and `.rpm` packages via our [public package
+repository](https://download.konghq.com/). Decide whether you want your
+Dockerfile to
+
+1. Download the desired package from https://download.konghq.com, or
+2. Download the desired package from another package repository you specify, or
+3. Install the desired package locally from disk.
+
+If you choose 1 or 2, run the command `touch kong.rpm` in the directory to which
+your Dockerfile will download the file; this guarantees that the downloaded file
+will have the correct user, groups, and permissions. 
+
+If you choose 2 or 3, then download the package you wish to install and put it
+in the desired location.
+
+### Write a Dockerfile to install the Kong Gateway package
+Use the template below to create your Dockerfile. Angle brackets (`<>`) indicate
+values that you need to provide. Comments that start "# Uncomment" indicate that
+you need to uncomment lines relevant to your context.
+
+The template is based upon the Dockerfiles in the the [docker-kong github
+repository](https://github.com/Kong/docker-kong/).
+
+```
+FROM <your-base-image> 
```

Comment:
Replicating dockerfile contents over here is a bit problematic IMO; this will very easily become out of sync with the source of truth (dockerfiles themselves).